rosbag2_compression/src/rosbag2_compression/zstd_compressor.cpp

```diff
@@ -71,7 +71,8 @@ std::vector<uint8_t> get_input_buffer(const std::string & uri)
     throw std::runtime_error{errmsg.str()};
   }
 
-  const auto decompressed_buffer_length = rcutils_get_file_size(uri.c_str());
+  const auto file_path = rcpputils::fs::path{uri};
+  const auto decompressed_buffer_length = file_path.exists() ? file_path.file_size() : 0;
```

Comment:
you might want to use `0u` here and below. You've done this in the change before as well.